content/v2.0/reference/flux/stdlib/contrib/influxdb/select.md

```diff
@@ -0,0 +1,180 @@
+---
+title: influxdb.select() function
+description: >
+  The `influxdb.select()` function is an alternate implementation of `from()`, `range()`, `filter()`
+  and `pivot()` that returns pivoted query results and masks the `_start` and `_stop` column
+  Results are similar to those returned by InfluxQL `SELECT` statements.
+menu:
+  v2_0_ref:
+    name: influxdb.select
+    parent: contrib_influxdb
+weight: 202
+v2.0/tags: [functions, package, query]
+related:
+  - /v2.0/reference/flux/stdlib/built-in/inputs/from/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/range/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/filter/
+  - /v2.0/reference/flux/stdlib/built-in/transformations/pivot/
```

Comment:
Is there a reference in the documentation to `v1.fieldsAsCols()`? It might be more helpful to link to.

Reply:
Good call. There is. I added it to the list.